reana_job_controller/kubernetes_job_manager.py

```diff
@@ -191,3 +201,42 @@ def add_shared_volume(self, job):
         job['spec']['template']['spec']['containers'][0][
             'volumeMounts'].append(volume_mount)
         job['spec']['template']['spec']['volumes'].append(volume)
+
+    def _add_krb5_sidecar_container(self, job, secrets_volume_mount):
+        """Add  sidecar container for a job."""
+        krb5_config_map_name = 'krb5-conf'
+        ticket_cache_volume = {
+            'name': 'krb5-cache',
+            'emptyDir': {}
+        }
+        krb5_config_volume = {
+            'name': 'krb5-conf',
+            'configMap': {'name': krb5_config_map_name}
+        }
+        volume_mounts = [
+            {
+                'name': ticket_cache_volume['name'],
+                'mountPath': current_app.config['KRB5_TOKEN_CACHE_LOCATION']
+            },
+            {
+                'name': krb5_config_volume['name'],
+                'mountPath': '/etc/krb5.conf',
+                'subPath': 'krb5.conf'
+            }
+        ]
+        keytab_file = os.environ.get('HTCONDORCERN_KEYTAB')
+        cern_user = os.environ.get('CERN_USER')
+        krb5_container = {
+            'image': current_app.config['KRB5_CONTAINER_IMAGE'],
+            'command': ['kinit', '-kt',
```

Comment:
After investigating a bit with @roksys, we found out that we could set the ticket cache location from the command line, like follows:
```
$ export KRB5CCNAME=DIR:/afs/cern.ch/user/r/rodrigdi/megatest && kinit
```
This would require to set the `KRB5CCNAME` environment variable on user's job containers.

To be addressed after merging this PR.